Freshli.Web/Views/Home/Privacy.cshtml

```diff
@@ -0,0 +1,6 @@
+﻿@{
+    ViewData["Title"] = "Privacy Policy";
+}
+<h1>@ViewData["Title"]</h1>
+
+<p>Use this page to detail your site's privacy policy.</p>
```

Comment:
Can you create an issue to update this page?

Reply:
Created issue #57.